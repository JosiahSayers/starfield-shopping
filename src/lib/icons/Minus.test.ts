import { describe, afterEach, it, expect } from 'vitest';
import { cleanup, render } from '@testing-library/svelte'
import Minus from './Minus.svelte'

describe('Minus', () => {
  afterEach(() => cleanup());

  it('mounts', () => {
    const { container } = render(Minus);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
