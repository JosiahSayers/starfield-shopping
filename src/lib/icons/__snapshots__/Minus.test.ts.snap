// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Minus > mounts 1`] = `"<div><svg xmlns=\\"http://www.w3.org/2000/svg\\" fill=\\"none\\" viewBox=\\"0 0 24 24\\" stroke-width=\\"1.5\\" stroke=\\"currentColor\\" class=\\"w-6 h-6\\"><path stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" d=\\"M19.5 12h-15\\"></path></svg></div>"`;
